scripts/bundle.js

```diff
@@ -65,14 +65,15 @@ function bundleJs(basePath, cfg, outfile) {
         'esbuild',
         '--bundle',
         `--platform=${cfg.platform || 'node'}`,
-        `--target=${cfg.target || 'es2018'}`,
+        `--target=${cfg.target || 'es2020'}`,
         `--format=${cfg.format || 'cjs'}`,
         ...Object.entries(cfg.env || {}).map(defineEnv),
         ...Object.entries(DefaultEnvVars).map(defineEnv),
         ...(cfg.external || []).map((c) => `--external:${c}`),
         `--outfile=${outfile}`,
         joinPath(basePath, cfg.entry),
     ];
+    console.log(buildCmd);
```

Comment:
`console.log` forgot to remove

Reply:
I kinda like the build command being logged :D